collect_app/src/main/java/org/odk/collect/android/fragments/ShowQRCodeFragment.java

```diff
@@ -117,72 +139,54 @@ private void addPasswordStatusString() {
         } else {
             status = getString(R.string.qrcode_without_passwords);
         }
-        editQRCode.setText(status);
+        tvPasswordWarning.setText(status);
     }
 
-    private void updateShareIntent(Bitmap qrCode) throws IOException {
-
-        // Send an intent to share saved image
+    private void updateShareIntent() {
+        // Initialize the intent to share qr code
```

Comment:
`qr` should probably be `QR`. Interesting that the patent calls it “QR code”, but the Denso web site, http://www.qrcode.com/en/history/, calls it “QR Code”.